packages/graph/hash_graph/lib/graph/src/datastore/postgres/database_type.rs

```diff
@@ -3,9 +3,14 @@ use crate::types::{
     VersionedUri,
 };
 
+/// A type, which is stored in a [`PostgresDatabase`].
```

Comment:
I don't think we have a good term for them yet but what do you mean by "type" here. As in a 'Type' from the Block Protocol Graph Service Type System? 😂

Reply:
(I'm not just being pedantic, it's unclear to me if we'd use this trait for entities for example)

Reply:
Literally a `type` which implements this trait 🙃

Reply:
With this PR we made the database unaware of the concept of `DataType`s, `EntityType`s, or `PropertyType`s.

Reply:
How about something like: 

> Provides an abstraction over elements of the Type System stored in the Database.

Reply:
This would imply, that it's only usable for the type system, but I expect to also implement this for entities.

Reply:
What about the term "entry"?

Reply:
> but I expect to also implement this for entities

Right and this is why I think these docs are important. I don't think the trait _will_ work for entities. Entities do not have a `VersionedUri`